packages/@uppy/aws-s3/src/index.ts

```diff
@@ -280,10 +280,10 @@ const defaultOptions = {
   allowedMetaFields: true,
   limit: 6,
   getTemporarySecurityCredentials: false as any,
-  shouldUseMultipart: ((file: UppyFile<any, any>) =>
-    file.size !== 0) as any as true, // TODO: Switch default to:
   // eslint-disable-next-line no-bitwise
-  // shouldUseMultipart: (file) => file.size >> 10 >> 10 > 100,
+  shouldUseMultipart: ((file: UppyFile<any, any>) =>
+    // eslint-disable-next-line no-bitwise
+    (file.size! >> 10) >> 10 > 100) as any as true,
```

Comment:
A bit too clever for my taste.

I think this is more clear `file.size > 100 * 1024 * 1024`